themes/mlibrary_new/css/style.css

```diff
@@ -994,12 +1036,19 @@ body>.navbar {
   }
 
   .exhibit-gallery-theme-item {
-  width: calc(100%);
+    width: calc(100%);
```

Comment:
You don't have to change it, but I'm wondering.  Is `calc(100%)` different from `100%` ?